src/SprykerEco/Zed/FactFinderSdk/Persistence/FactFinderSdkQueryContainer.php

```diff
@@ -35,6 +34,13 @@ class FactFinderSdkQueryContainer extends AbstractQueryContainer implements Fact
     public const STOCK_NEVER_OUTOFSTOCK_CONDITION = 'STOCK_NEVER_OUTOFSTOCK_CONDITION';
 
     /**
+     * @uses \Orm\Zed\Url\Persistence\Map\SpyUrlTableMap::COL_URL
+     */
+    protected const COLUMN_URL = 'spy_url.url';
+
+    /**
+     * {@inheritDoc}
+     *
```

Comment:
Please add to the method document block `@module Url`.

Reply:
Fixed.